Source/VMware.vSphereDSC/Tests/Integration/VMHostVss/VMHostVss.Integration.Tests.ps1

```diff
@@ -221,12 +224,6 @@ try {
                 $configuration = Get-DscConfiguration
 
                 # Assert
-                $configuration.Server | Should -Be $script:resourceWithRemoveVss.Server
-                $configuration.Name | Should -Be $script:resourceWithRemoveVss.Name
-                $configuration.Ensure | Should -Be $script:resourceWithRemoveVss.Ensure
-                $configuration.VssName | Should -Be $script:resourceWithRemoveVss.VssName
-                $configuration.NumPorts | Should -Be $script:resourceWithRemoveVss.NumPorts
-                $configuration.Mtu | Should -Be $script:resourceWithRemoveVss.Mtu
                 $configuration.Ensure | Should -Be $script:resourceWithRemoveVss.Ensure
```

Comment:
Here you should also check the Server, Name and VssName properties.

Reply:
Changed